lib/aws/session_store/dynamo_db/locking/pessimistic.rb

```diff
@@ -1,144 +0,0 @@
-module Aws::SessionStore::DynamoDB::Locking
```

Comment:
I'm a little skeptical about the removal of locking as a feature.  I don't think consistent reads solves quite the same problem.

Reply:
Talked offline. You are right I'll bring it back.